src/node_snapshotable.cc

```diff
@@ -130,16 +141,44 @@ void SnapshotBuilder::Generate(SnapshotData* out,
                             nullptr,
                             node::EnvironmentFlags::kDefaultFlags,
                             {});
+
       // Run scripts in lib/internal/bootstrap/
       {
         TryCatch bootstrapCatch(isolate);
-        v8::MaybeLocal<Value> result = env->RunBootstrapping();
+        MaybeLocal<Value> result = env->RunBootstrapping();
         if (bootstrapCatch.HasCaught()) {
           PrintCaughtException(isolate, context, bootstrapCatch);
         }
         result.ToLocalChecked();
       }
 
+      // Run the entry point file
+      if (!entry_file.empty()) {
+#if HAVE_INSPECTOR
+        env->InitializeInspector({});
+#endif
+
+        TryCatch bootstrapCatch(isolate);
+        // TODO(joyee): we could use the result for something special, like
+        // setting up initializers that should be invoked at snapshot
+        // dehydration.
```

Comment:
```suggestion
        // TODO(joyeecheung): we could use the result for something
        // special, like setting up initializers that should be invoked
        // at snapshot dehydration.
```
For consistency since all (or most) TODO comments have full usernames.